toolkit/scripts/toolchain.mk

```diff
@@ -19,8 +19,7 @@ toolchain_downloads_logs_dir = $(toolchain_logs_dir)/downloads
 toolchain_downloads_manifest = $(toolchain_downloads_logs_dir)/download_manifest.txt
 toolchain_log_tail_length = 20
 populated_toolchain_chroot = $(toolchain_build_dir)/populated_toolchain
-toolchain_sources_dir = $(populated_toolchain_chroot)/usr/src/mariner/SOURCES
```

Comment:
Removed `toolchain_sources_dir` because it just wasn't used anywhere in the script.